app/routes/lending/components/LendingRequestAdmin.tsx

```diff
@@ -0,0 +1,177 @@
+import dayGridPlugin from '@fullcalendar/daygrid';
+import interactionPlugin from '@fullcalendar/interaction';
+import FullCalendar from '@fullcalendar/react';
+import timeGridPlugin from '@fullcalendar/timegrid';
+import { LoadingIndicator } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { Helmet } from 'react-helmet-async';
+import { Link, useParams } from 'react-router-dom';
+import {
+  fetchLendingRequest,
+  fetchLendingRequestsForLendableObject,
+} from 'app/actions/LendingRequestActions';
+import {
+  Content,
+  ContentMain,
+  ContentSection,
+  ContentSidebar,
+} from 'app/components/Content';
+import InfoList from 'app/components/InfoList';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import { FromToTime } from 'app/components/Time';
+import {
+  selectLendingRequestById,
+  selectLendingRequestsByLendableObjectId,
+} from 'app/reducers/lendingRequests';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+
+type Params = {
+  lendingRequestId: string;
+};
+
+const LendingRequestAdmin = () => {
+  const { lendingRequestId } = useParams<Params>();
+  const dispatch = useAppDispatch();
+
+  usePreparedEffect(
+    'fetchRequest',
+    () => dispatch(fetchLendingRequest(lendingRequestId)),
+    [],
+  );
```

Comment:
```suggestion
  usePreparedEffect(
    'fetchLendingRequest',
    () => lendingRequestId && dispatch(fetchLendingRequest(lendingRequestId)),
    [lendingRequestId],
  );
```

id should be more specific to avoid dups